recipes/libcpuid/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "libcpuid" %}
```

Comment:
Does this library guarantee any ABI stability? It definitely needs a `run_exports` entry.

Reply:
Thanks for the comments and suggestions, much appreciated. I've added a `run_exports` for `libcpuid`. This is a little tricky, since I'm not the developer of that package, I'm just maintaining it here since it's an optional (but useful) dependency of our package. As such, I've not set any range for the pinning.